tools/block_cache_trace_analyzer.cc

```diff
@@ -27,6 +27,9 @@ DEFINE_string(
     "cache_name,num_shard_bits,cache_capacity_1,...,cache_capacity_N. "
     "cache_name is lru. cache_capacity can be xK, xM or xG "
     "where x is a positive number.");
+DEFINE_int32(block_cache_trace_downsample_ratio, 1,
```

Comment:
Maybe explicitly call out the meaning of this value, e.g. sample one in every `block_cache_trace_downsample_ratio` accesses?

Reply:
Sure. I updated the comments.